filesystems/s3/src/main/java/org/lerch/s3fs/S3FileSystemProvider.java

```diff
@@ -654,7 +765,8 @@ public boolean isOpen(S3FileSystem s3FileSystem) {
      * only 4 testing
      */
 
-    protected static ConcurrentMap<String, S3FileSystem> getFilesystems() {
+    @VisibleForTesting
```

Comment:
Is this just a marker annotation or does it do something particular?

Reply:
It's just a marker to say that the only reason it needs to be this visible is for testing purposes. I believe it came from the original code that Emil Lerch forked. It could be removed if you want.

Reply:
I think it's fine as it is, since it's not the only dependency on Guava in this class